StringTheory/UI/StringList.xaml

```diff
@@ -86,6 +86,12 @@
                 <DataGridTextColumn Width="Auto" ElementStyle="{StaticResource StatisticStyle}" Header="LOH%"   Binding="{Binding LohPercent,  Converter={x:Static local:PercentageConverter.Default}}" local:DataGridExtensions.SortDesc="True" />
             </DataGrid.Columns>
 
+            <DataGrid.Resources>
+                <Style TargetType="DataGridRow">
+                    <EventSetter Event="MouseDoubleClick" Handler="DataGridRow_DoubleClick"/>
```

Comment:
Adding event handlers (`DataGridRow_DoubleClick` here) in WPF is generally frowned upon. When the UI wants to invoke code in the model, binding to an `ICommand` is usually preferrable.

We already have a command for showing referrers, and it's bound to the context menu item. You can reuse it here, and add an entry to `Grid.InputBindings` on line 75. Something like:

```xml
<MouseBinding Gesture="LeftDoubleClick" Command="{Binding ShowReferrersCommand}" />
```

With that, we shouldn't have to add any code behind, which is a benefit of commands.